### ISSUE TITLE
Shadowgraphy plugin is not excluded when openPMD is not available

### ISSUE THREAD

**Issue by PrometheusPi** [ComputationalRadiationPhysics/picongpu#4897]
When compiling PIConGPU with fftw3 avaialbe but not openPMD-api, the compiler will fail with the following error:
```
In file included from /p/project/___________/_______/picongpu/include/picongpu/../picongpu/plugins/shadowgraphy/Shadowgraphy.hpp:26,
                 from /p/project/___________/_______/picongpu/include/picongpu/../picongpu/plugins/PluginController.hpp:47,
                 from /p/project/___________/_______/picongpu/include/picongpu/../picongpu/unitless/starter.unitless:23,
                 from /p/project/___________/_______/picongpu/include/picongpu/../picongpu/simulation_defines.hpp:58,
                 from /p/project/___________/_______/picongpu/include/picongpu/main.cpp:23:
/p/project/___________/_______/picongpu/include/picongpu/../picongpu/plugins/common/openPMDAttributes.hpp:27:10: fatal error: openPMD/openPMD.hpp: No such file or directory
   27 | #include <openPMD/openPMD.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

It appears to be caused by a missing include guard.

**Comment by psychocoderHPC** [comment]
The guards are missing in `PluginController.hpp`

**Comment by psychocoderHPC** [comment]
fixed with #4898
